docs/src/pages/[platform]/theming/authenticator-theming.mdx

```diff
@@ -0,0 +1,69 @@
+import { Fragment } from '@/components/Fragment';
+import { Example, ExampleCode } from '@/components/Example';
+import { Authenticator } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import {
+  AuthenticatorButtonClassExample,
+  AuthenticatorClasses,
+} from './examples';
+
+A Theme is a structured collection of design decisions that change the appearance of a UI library. An Amplify UI theme allows application wide visual changes through the use of targeted CSS classes and variables.
+
+## Getting started
+
+<Fragment>
+  {({ platform }) =>
+    import(`../components/authenticator/quick-start-example.${platform}.mdx`)
+  }
+</Fragment>
+
+### Classes
+
+You can override the Authenticator's `amplify-*` classes to Theme the authenticator. For example to style all of the buttons in the Authenticator you could target the `amplify-button` class.
+
+<Example className="authenticator-theme-class-example">
+  <AuthenticatorButtonClassExample />
+
+  <ExampleCode>
+
+```css
+.amplify-button[data-variation='link']:hover {
+  background: none;
```

Comment:
Nit, when I was testing this example, I couldn't see `cursor: pointer` when I was hovering the link. Was something overridden here?